bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/handler/EaseeSiteHandler.java

```diff
@@ -174,6 +175,15 @@ public void dispose() {
         webInterface.dispose();
     }
 
+    @Override
+    protected void updateConfiguration(Configuration configuration) {
+        super.updateConfiguration(configuration);
+        getChildChargerHandlers().forEach((name, handler) -> {
+            logger.debug("notify {}: config update", name);
+            handler.reInit();
+        });
```

Comment:
Looks like a bad approach when I see that reInit method is doing a dispo and initialize of the thing handler.
Normally the management of thing handler lifecycle should be done only by the thing manager.
By changing the thing configuration, your bridge thing status should change and any of these changes are notified to each child thing handlers through the method bridgeStatusChanged.

Reply:
I changed it according to your suggestion.